src/main/java/org/apache/commons/io/ByteOrderParser.java

```diff
@@ -25,38 +25,26 @@
  *
  * @since 2.6
  */
-public final class ByteOrderUtils {
-
-    private static final Locale ComparisonLocale = Locale.ROOT;
-
-    /**
-     * Big endian.
-     */
-    public static final String BIG_ENDIAN = "Big";
-
-    /**
-     * Little endian.
-     */
-    public static final String LITTLE_ENDIAN = "Little";
+public final class ByteOrderParser {
 
     /**
      * ByteOrderUtils is a static utility class, so prevent construction with a private constructor.
      */
-    private ByteOrderUtils() {
+    private ByteOrderParser() {
     }
 
     /**
      * Parses the String argument as a {@link ByteOrder}, ignoring case.
```

Comment:
I think the pull request changes this, or I'm I wrong? Maybe we should keep the case insensitivity?

Reply:
You're right. I've changed the documentation. Let's make it restrictive and see if users want to have a more relaxed handling.